bloom-housing/bloom
# feat: add marked and flagged as duplicate to all applications table

# Pull Request Template

## Issue Overview

This PR addresses #3103


## Description

Added two columns `flagged as duplicate`, and `marked as duplicate` to All Aplications.
Marked as duplicate has its own field. Flagged as duplicate is when reviewStatus equals 'duplicate'

## How Can This Be Tested/Reviewed?

Go to applications tab in listing. In All Application table scroll to the right. Last two columns should be `Flagged as Duplicate`, and `Marked as Duplicate`. On null / undefined state they should display n/a (not sure if possible)

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
